paddle/scripts/paddle_build.sh

```diff
@@ -367,7 +367,12 @@ function run_test() {
     Running unit tests ...
     ========================================
 EOF
-        ctest --output-on-failure
+        if [ ${TESTING_DEBUG_MODE:-OFF} == "ON" ] ; then
+            ctest -V
```

Comment:
Why in daily CI we need all the logs? fail log should be enough I think.

Reply:
Only failed unit test will be shown, but the ```Timeout``` will not.

Reply:
Well, we are running ctest in parallel, and there are many tests outputing a lot, the log may be messed up is we turn it on, may not help to debug.

Anyway I'll approve this since it's an option.